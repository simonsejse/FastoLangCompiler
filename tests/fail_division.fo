fun int main() = 
  let a = 20 in
  let b = 4 in
  let c = 0 in
  let result1 = a / b in
  let result2 = a / c in  // This should raise "Division by zero"
  write(result1)
