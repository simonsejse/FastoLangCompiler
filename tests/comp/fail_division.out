Runtime error at line 6: division by zero
